### ISSUE TITLE
[BUG] After upgrading to 2.0.0 get 301 redirect loop from front

### ISSUE THREAD

**Issue by mglants** [Mailu/helm-charts#352]
**Describe the bug**
when accessing web interface, got redirected loop

**Environment**
 - Talos Kubernetes
 -  Ingress nginx
 - Mailu 2024.06.3

**Additional context**

Imap works normally 
```
92.168.51.254 - - [25/Jun/2024:03:51:32 +0300] "GET / HTTP/1.1" 301 162 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36"
192.168.51.254 - - [25/Jun/2024:03:51:32 +0300] "GET / HTTP/1.1" 301 162 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36"
192.168.51.254 - - [25/Jun/2024:03:51:32 +0300] "GET / HTTP/1.1" 301 162 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/126.0.0.0 Safari/537.36"
```
Values.yaml
```
    hostnames:
      - mail.ur30.ru
      - webmail.ur30.ru

ingress:
      ## @param ingress.enabled Enable external ingress
      enabled: true
      ingressClassName: nginx

      annotations:
        external-dns.alpha.kubernetes.io/target: ingress.ur30.ru
        cert-manager.io/cluster-issuer: letsencrypt-prod
      tls: true

      ## @param ingress.realIpHeader SrealIpFromets the value of `REAL_IP_HEADER` environment variable in the `front` pod
      realIpHeader: X-Forwarded-For

      ## @param ingress.realIpFrom Sets the value of `REAL_IP_FROM` environment variable in the `front` pod
      realIpFrom: 10.244.0.0/16,10.96.0.0/12
```

**Comment by Sacerdoss** [comment]
I have had the same error after upgrading. Turns out the nginx configuration will enable the https redirect with default values. We have set `ingress.tlsFlavorOverride: mail` which will disable that behavior so webmailer can be accessed through the kuebrnetes ingress controller on port 80 again.

**Comment by mglants** [comment]
Yep that worked, should it be the default for ingress?

**Comment by Sacerdoss** [comment]
> Yep that worked, should it be the default for ingress?

It probably should if `ingress.tls` is set to `true`

**Comment by fastlorenzo** [comment]
Fixed in #354

**Comment by taoyonggang** [comment]
I am using the latest version. But the bug is exist.

* Request completely sent off
< HTTP/1.1 302 Found
< Server: nginx
< Date: Tue, 22 Oct 2024 13:43:00 GMT
< Content-Type: text/html; charset=UTF-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Set-Cookie: roundcube_sessid=b63a4c62e651e034bae64a9ca0632500; path=/; HttpOnly
< Expires: Tue, 22 Oct 2024 13:43:00 GMT
< Last-Modified: Tue, 22 Oct 2024 13:43:00 GMT
< Cache-Control: private, no-cache, no-store, must-revalidate, post-check=0, pre-check=0
< Pragma: no-cache
< X-Frame-Options: sameorigin
< Set-Cookie: roundcube_sessauth=-del-; expires=Tue, 22 Oct 2024 13:42:00 GMT; Max-Age=0; path=/; HttpOnly
< Location: sso.php
< X-Download-Options: noopen
< X-Robots-Tag: none
< X-Permitted-Cross-Domain-Policies: none
< Referrer-Policy: no-referrer
* Ignoring the response-body
< 
* Connection #0 to host mailu-webmail left intact
* Issue another request to this URL: 'http://mailu-webmail/sso.php'
* Re-using existing connection with host mailu-webmail
> GET /sso.php HTTP/1.1
> Host: mailu-webmail
> User-Agent: curl/8.10.1
> Accept: */*
> 
* Request completely sent off
< HTTP/1.1 302 Found
< Server: nginx
< Date: Tue, 22 Oct 2024 13:43:00 GMT
< Content-Type: text/html; charset=UTF-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Set-Cookie: roundcube_sessid=270630c416d53548fedd0184314c2418; path=/; HttpOnly
< Expires: Tue, 22 Oct 2024 13:43:00 GMT
< Last-Modified: Tue, 22 Oct 2024 13:43:00 GMT
< Cache-Control: private, no-cache, no-store, must-revalidate, post-check=0, pre-check=0
< Pragma: no-cache
< X-Frame-Options: sameorigin
< Set-Cookie: roundcube_sessauth=-del-; expires=Tue, 22 Oct 2024 13:42:00 GMT; Max-Age=0; path=/; HttpOnly
< Location: sso.php
< X-Download-Options: noopen
< X-Robots-Tag: none
< X-Permitted-Cross-Domain-Policies: none
< Referrer-Policy: no-referrer
* Ignoring the response-body
< 
* Connection #0 to host mailu-webmail left intact
* Issue another request to this URL: 'http://mailu-webmail/sso.php'
* Re-using existing connection with host mailu-webmail
> GET /sso.php HTTP/1.1
> Host: mailu-webmail
> User-Agent: curl/8.10.1
> Accept: */*
> 
* Request completely sent off
< HTTP/1.1 302 Found
< Server: nginx
< Date: Tue, 22 Oct 2024 13:43:00 GMT
< Content-Type: text/html; charset=UTF-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Set-Cookie: roundcube_sessid=e36cc7e02e35739d278b0eac844b3bba; path=/; HttpOnly
< Expires: Tue, 22 Oct 2024 13:43:00 GMT
< Last-Modified: Tue, 22 Oct 2024 13:43:00 GMT
< Cache-Control: private, no-cache, no-store, must-revalidate, post-check=0, pre-check=0
< Pragma: no-cache
< X-Frame-Options: sameorigin
< Set-Cookie: roundcube_sessauth=-del-; expires=Tue, 22 Oct 2024 13:42:00 GMT; Max-Age=0; path=/; HttpOnly
< Location: sso.php
< X-Download-Options: noopen
< X-Robots-Tag: none
< X-Permitted-Cross-Domain-Policies: none
< Referrer-Policy: no-referrer
* Ignoring the response-body
< 
* Connection #0 to host mailu-webmail left intact
* Issue another request to this URL: 'http://mailu-webmail/sso.php'
* Re-using existing connection with host mailu-webmail
> GET /sso.php HTTP/1.1
> Host: mailu-webmail
> User-Agent: curl/8.10.1
> Accept: */*
> 
* Request completely sent off
< HTTP/1.1 302 Found
< Server: nginx
< Date: Tue, 22 Oct 2024 13:43:01 GMT
< Content-Type: text/html; charset=UTF-8
< Transfer-Encoding: chunked
< Connection: keep-alive
< Set-Cookie: roundcube_sessid=fd6ab19b0efdca682c72114e96e76551; path=/; HttpOnly
< Expires: Tue, 22 Oct 2024 13:43:01 GMT
< Last-Modified: Tue, 22 Oct 2024 13:43:01 GMT
< Cache-Control: private, no-cache, no-store, must-revalidate, post-check=0, pre-check=0
< Pragma: no-cache
< X-Frame-Options: sameorigin
< Set-Cookie: roundcube_sessauth=-del-; expires=Tue, 22 Oct 2024 13:42:01 GMT; Max-Age=0; path=/; HttpOnly
< Location: sso.php
< X-Download-Options: noopen
< X-Robots-Tag: none
< X-Permitted-Cross-Domain-Policies: none
< Referrer-Policy: no-referrer
* Ignoring the response-body
< 
* Connection #0 to host mailu-webmail left intact
* Maximum (50) redirects followed

The webmail can't work.
